admin/merge_pr.py

```diff
@@ -279,6 +285,26 @@ def transform(x):
     return None
 
 
+def jenkins_info_from_response(project):
+    """
+    Get the Jenkins job information from the Jenkins API response.
+
+    :param dict project: the API response as a dict.
+    :return (JenkinsResults, dict): The first element will
```

Comment:
Yea, since working here I've started just writing 2 attribute PClasses for things like this, but our requirements for docstring sometimes make that have more overhead than it probably should.